pkg/auth/handler/webapp/panic_middleware.go

```diff
@@ -29,8 +30,16 @@ func (m *PanicMiddleware) Handle(next http.Handler) http.Handler {
 				viewmodels.Embed(data, baseViewModel)
 				m.Renderer.RenderHTML(w, r, TemplateWebFatalErrorHTML, data)
 
-				// Rethrow
-				panic(err)
+				// Only rethrow if the err is NOT api error.
+				rethrow := true
+				if err, ok := err.(error); ok {
+					if apierrors.IsAPIError(err) {
+						rethrow = false
+					}
+				}
+				if rethrow {
+					panic(err)
+				}
```

Comment:
Would like to know why do we need this change?

Reply:
This is because we now detect `interaction.ErrNoAuthenticator` in `select_account.go` before we start interaction. We use `panic` to report such error. If we do not have this change, we will see panic log about `interaction.ErrNoAuthenticator`, which is actually an expected error.